public-ui/src/app/data-browser/base-react/base-react.wrapper.tsx

```diff
@@ -1,32 +1,25 @@
 import {
     AfterViewInit,
-    Component,
+    Directive,
     ElementRef,
-    Injector,
     OnChanges,
     OnDestroy,
-    SimpleChanges,
     ViewChild,
-    ViewEncapsulation
 } from '@angular/core';
+import * as fp from 'lodash/fp';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-const containerElementName = 'myReactComponentContainer';
 
-@Component({
-    // tslint:disable-next-line: component-selector
-    selector: 'base-react-wrapper',
-    template: `<span #${containerElementName}></span>`,
-    styleUrls: ['../../styles/template.css'],
-    encapsulation: ViewEncapsulation.None,
-})
-  // tslint:disable-next-line: component-class-suffix
+// Add empty directive decorator to base class per Angular docs:
+// https://angular.io/guide/migration-undecorated-classes
+@Directive()
```

Comment:
Used directive instead of component since it doesn't require any arguments and is recommended in [Angular docs](https://angular.io/guide/migration-undecorated-classes)